hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/NNThroughputBenchmark.java

```diff
@@ -1346,9 +1376,9 @@ void printResults() {
   class ReplicationStats extends OperationStatsBase {
     static final String OP_REPLICATION_NAME = "replication";
     static final String OP_REPLICATION_USAGE = 
-        "-op replication [-datanodes T] [-nodesToDecommission D] " +
+        "-op replication [-datanodes T] [-nodesToDecommission N] " +
```

Comment:
we can't change the existing commands, that is an incomatible change, whatever needs to be done should be done for new commands only.

btw. what does this letter denote?

Reply:
I think D denote dir.

Reply:
Can we both -nodesToDecommission and -baseDirName use D?

Reply:
If it has no physical meaning, then yes, can check if other alphabets are not used, so can explore using them P as in Path or B as in BaseDir or some similar, else use whichever you like :-)

Reply:
P and B has also been used, such as -blocksPerReport B and -filesPerDir P. And other alphabets cannot be related to directory I think. Could I keep using D?

Reply:
ok, just double check they don't have any specific usage apart from just being alphabets in the description

Reply:
ok. Only -nodesToDecommission used the D alphabet.